test/test.pretty/10_uMAX.kicad_mod

```diff
@@ -0,0 +1,42 @@
+(module 10_uMAX (layer F.Cu) (tedit 6085D3C9)
+  (descr "3x3mm uMAX, 10 leads")
+  (tags "u10 uMAX 90-0330 21-0061")
+  (attr smd)
+  (fp_text reference REF** (at 0 -2.54) (layer F.SilkS)
+    (effects (font (size 1 1) (thickness 0.15)))
+  )
+  (fp_text value 10_uMAX (at 0 2.54) (layer F.Fab)
+    (effects (font (size 1 1) (thickness 0.15)))
+  )
+  (fp_line (start -2.45 1.5) (end -2.45 -1.5) (layer F.Fab) (width 0.1))
+  (fp_line (start 2.45 1.5) (end -2.45 1.5) (layer F.Fab) (width 0.1))
+  (fp_line (start 2.45 -1.5) (end 2.45 1.5) (layer F.Fab) (width 0.1))
+  (fp_line (start -2.45 -1.5) (end 2.45 -1.5) (layer F.Fab) (width 0.1))
+  (fp_circle (center -2.14 -1.5) (end -2.09 -1.5) (layer F.SilkS) (width 0.12))
+  (fp_line (start -1.25 -1.25) (end 1.25 -1.25) (layer F.SilkS) (width 0.12))
+  (fp_line (start 1.25 -1.25) (end 1.25 1.25) (layer F.SilkS) (width 0.12))
+  (fp_line (start 1.25 1.25) (end -1.25 1.25) (layer F.SilkS) (width 0.12))
+  (fp_line (start -1.25 1.25) (end -1.25 -1.25) (layer F.SilkS) (width 0.12))
+  (fp_line (start -2.9 -1.5) (end 2.9 -1.5) (layer F.CrtYd) (width 0.05))
+  (fp_line (start 2.9 -1.5) (end 2.9 1.5) (layer F.CrtYd) (width 0.05))
+  (fp_line (start 2.9 1.5) (end -2.9 1.5) (layer F.CrtYd) (width 0.05))
+  (fp_line (start -2.9 1.5) (end -2.9 -1.5) (layer F.CrtYd) (width 0.05))
+  (fp_text user %R (at 0 -2.54) (layer F.Fab)
+    (effects (font (size 1 1) (thickness 0.15)))
+  )
+  (pad 1 smd rect (at -2.14 -1) (size 1.4 0.28) (layers F.Cu F.Paste F.Mask))
+  (pad 2 smd rect (at -2.14 -0.5) (size 1.4 0.28) (layers F.Cu F.Paste F.Mask))
+  (pad 3 smd rect (at -2.14 0) (size 1.4 0.28) (layers F.Cu F.Paste F.Mask))
+  (pad 4 smd rect (at -2.14 0.5) (size 1.4 0.28) (layers F.Cu F.Paste F.Mask))
+  (pad 5 smd rect (at -2.14 1) (size 1.4 0.28) (layers F.Cu F.Paste F.Mask))
+  (pad 6 smd rect (at 2.14 1) (size 1.4 0.28) (layers F.Cu F.Paste F.Mask))
+  (pad 7 smd rect (at 2.14 0.5) (size 1.4 0.28) (layers F.Cu F.Paste F.Mask))
+  (pad 8 smd rect (at 2.14 0) (size 1.4 0.28) (layers F.Cu F.Paste F.Mask))
+  (pad 9 smd rect (at 2.14 -0.5) (size 1.4 0.28) (layers F.Cu F.Paste F.Mask))
+  (pad 10 smd rect (at 2.14 -1) (size 1.4 0.28) (layers F.Cu F.Paste F.Mask))
+  (model ${KISYS3DMOD}/test.3dshapes/10_uMAX.wrl
```

Comment:
This 3D model does not exist, but it is not worth renaming the test libraries to fix this since this is just a prototype.